shells/webextension/icons/canary.svg

```diff
@@ -0,0 +1,69 @@
+<svg id="Outdated" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024">
```

Comment:
Using https://jakearchibald.github.io/svgomg/ to optimize the svg.

```
<svg id="Outdated" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024"><defs><style>.cls-1{fill:#ffba00}.cls-2{fill:#fff}</style></defs><rect class="cls-1" width="1024" height="1024" rx="96" ry="96" id="Background"/><g id="Rings"><path class="cls-2" d="M510 325c-249 0-450 83-450 185s201 185 450 185 450-83 450-185-201-185-450-185zm0 330c-225 0-407-65-407-145s182-145 407-145 408 65 408 145-183 145-408 145z"/><path class="cls-2" d="M670 417C546 202 373 69 285 120s-59 267 65 482 297 349 385 298 59-267-65-483zM384 583c-112-195-147-386-78-426s217 86 329 280 148 385 78 425-216-85-329-279z"/><g id="BLOCKER"><path class="cls-1" transform="rotate(39 725.584 779.096)" d="M564 572h315v397.12H564z"/><path class="cls-1" transform="rotate(21 834.313 556.928)" d="M685 379h283v360.81H685z"/></g><path class="cls-2" d="M670 603c124-215 153-431 65-482s-261 81-385 296-153 432-65 483 261-82 385-297zM384 437c113-194 260-320 330-280s34 231-78 425-260 321-330 281-34-231 78-426z"/></g><circle class="cls-2" cx="512" cy="512" r="80" id="Circle"/><path transform="translate(585 585)" d="M372.888 3.417C370.942 1.471 367.649.678 361.52.678c-27.119 0-107.25 15.936-169.59 52.773l-2.123 1.254-9.154 75.342-21.625-54.922-5.274 4.101c-8.973 6.976-17.139 14.13-24.271 21.262-34.747 34.747-58.945 73.784-68.136 109.918-7.562 29.727-4.785 56.001 7.688 75.652L4.393 350.699c-5.858 5.858-5.858 15.355 0 21.213 2.929 2.929 6.768 4.394 10.606 4.394s7.678-1.464 10.606-4.394l64.652-64.652c12.327 7.798 27.362 11.868 44.258 11.869h.003c43.844 0 95.348-26.356 141.297-72.306.854-.854 1.701-1.728 2.589-2.646l5.212-5.384-60.929-23.991 87.707-10.657 1.275-1.917c12.748-19.177 24.504-40.653 34.943-63.832l2.154-4.784-44.544-17.539 54.349-6.604 1.057-2.866c7.921-21.486 13.639-44.894 16.101-65.91 1.65-14.06 2.494-31.941-2.841-37.276z" fill="#202020" id="Feather"/></svg>
```
`1.16k → 1014 bytes 85.07%`